### PR TITLE
OHRI-120 - Patient List Widget

### PR DESCRIPTION
**Pull request by pirupius**
Screenshot 2021-07-07 at 15 09 44

### CODE REVIEW

**Comment by larslemos** [comment]
@pirupius  Your PR has old code, that's why is failing the Build. Please pull the latest.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "openmrs-ohri-app",
-  "version": "0.6.2",
+  "name": "openmrs-esm-ohri-app",
+  "version": "0.9.0",
```
Comment (samuelmale) [inline package.json:3]: @pirupius why did you touch the above

File: src/hts/patient-list/patient-list.component.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  useEffect(() => {
+    setIsLoading(true);
+    loadPatients(nextOffSet, pageSize);
+  }, [page, pageSize]);
+
+  const addNewPatient = () => navigate({ to: '${openmrsSpaBase}/patient-registration' });
+  const navigateToPatientDB = patientUuid => navigate({ to: '${openmrsSpaBase}/patient/' + `${patientUuid}/chart` });
+
+  function loadPatients(offSet: number, pageSize: number) {
+    return openmrsFetch(`/ws/fhir2/R4/Patient?_getpagesoffset=${offSet}&_count=${pageSize}`).then(({ data }) => {
```
Comment (samuelmale) [inline src/hts/patient-list/patient-list.component.tsx:43]: Create this function in ../api/api.ts

File: package.json (hunk 1 above)
Comment (pirupius) [reply package.json:3]: Reverting this

**Comment by samuelmale** [comment]
@pirupius is this still a WIP?

**Pull request merged**